qa/common/gen_xavier_trt_models

```diff
@@ -50,13 +50,13 @@
 ##
 ############################################################################
 
-TRITON_VERSION=22.07
+TRITON_VERSION=${TRITON_VERSION:=22.07}
 CUDA_DEVICE=0
 
-REPO_ARCH="xavier"
```

Comment:
We don't need `*_xavier` in the model directory name? Isn't that useful in differentiating the model stores for different platforms?

Reply:
This script designed to generate models, so that folder is required if you trying to differentiate the local directories. 
I can restore it back, but I removed it to keep some consistency. <br>
Important is the directory that we passing to the `test.sh` script.